configure.sh

```diff
@@ -43,6 +43,7 @@ usage() {
 }
 
 setup_opensource() {
+  mkdir -p "$(dirname ${PRIVATE_NETWORK_CONFIG})" # Ensure operation in case of a sparse checkout
```

Comment:
This is not needed if we always checkout everything for full builds (just make sure to include at least 100 commits and recursively checkout submodules)